### PR TITLE
Add reading-comprehension and nlvr-parser to routing

### LINKED ISSUE #176: Direct links for reading comprehension and nlvr parser 404
https://demo.allennlp.org/nlvr-parser
https://demo.allennlp.org/reading-comprehension

### PR DESCRIPTION
**Pull request by matt-gardner**
Fixes #176.

### CODE REVIEW

**Comment by matt-gardner** [comment]
This is live on staging, and it appears to be fixed.  It's unfortunate that we have to merge PRs to really test this - I was pretty sure this solved the problem when I tested it locally, but it'd be nice to be able to see a deployed version of a PR _before_ merging it.

File: app.py [hunk 1]
```diff
@@ -300,11 +300,13 @@ def info() -> Response:  # pylint: disable=unused-variable
     @app.route('/constituency-parsing')
     @app.route('/dependency-parsing')
     @app.route('/machine-comprehension')
```
Comment (jonborchardt) [inline app.py]: why are we keeping the old route? not sure the FE is set up to handle it...

File: app.py [hunk 2]
```diff
@@ -313,11 +315,13 @@ def info() -> Response:  # pylint: disable=unused-variable
     @app.route('/constituency-parsing/<permalink>')
     @app.route('/dependency-parsing/<permalink>')
     @app.route('/machine-comprehension/<permalink>')
```
Comment (jonborchardt) [inline app.py]: same

File: app.py (hunk 1 above)
Comment (matt-gardner) [reply app.py]: Because I didn't want to break anything.  I just checked what's on staging, and it looks like I didn't need to be worried.  Fixing this.

**Pull request merged**